plugins/Files/js/components/filelist.js

```diff
@@ -60,12 +60,27 @@ const FileList = ({files, selected, searchResults, path, showSearchField, action
 			/>
 		)
 	})
+
+	const fileListContents = (() => {
+		if (!hasPolled) {
+			return (
+				<h2> Loading files... </h2>
+			)
+		}
+		if (fileElements.size === 0) {
```

Comment:
would be nicer if we could just compare `fileElements === null` or `undefined` or something instead of needing a separate var.